code/game/machinery/computer/robot.dm

```diff
@@ -136,13 +138,21 @@
 	else if (href_list["magbot"])
 		if((issilicon(usr) && is_special_character(usr)) || IsAdminGhost(usr))
 			var/mob/living/silicon/robot/R = locate(href_list["magbot"])
-			if(istype(R) && !R.emagged && ((R.syndicate && R == usr)|| R.connected_ai == usr || IsAdminGhost(usr)) && !R.scrambledcodes && can_control(usr, R))
+			if(istype(R) && !R.emagged && ((R.syndicate && R == usr) || R.connected_ai == usr || IsAdminGhost(usr)) && !R.scrambledcodes && can_control(usr, R))
 				log_game("[key_name(usr)] emagged [R.name] using robotic console!")
-				message_admins("[key_name_admin(usr)] emagged cyborg [key_name_admin(R)]. using robotic console!")
+				message_admins("[key_name_admin(usr)] emagged cyborg [key_name_admin(R)] using robotic console!")
 				R.SetEmagged(1)
 				if(is_special_character(R))
 					R.verbs += /mob/living/silicon/robot/proc/ResetSecurityCodes
 
+	else if(href_list["convert"])
+		if(issilicon(usr) && is_special_character(usr))
+			var/mob/living/silicon/robot/R = locate(href_list["convert"])

```

Comment:
It doesn't let just ANYONE convert ANY robot, because it has proper checks both before and after, like `if(issilicon(usr) && is_special_character(usr))` and `if(istype(R) && !is_servant_of_ratvar(R) && is_servant_of_ratvar(usr) && R.connected_ai == usr)`
So if you pass all of those checks honestly you're supposed to be able to do it anyway